.github/workflows/tests.yml

```
@@ -55,6 +62,11 @@
       run: |
         pipenv run pylint "$(git ls-files '*.py')" || true
 
+    - name: Run frontend lint
+      if: "exists('app/package.json')"
```

actionlint (1.7.7)

undefined function "exists". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)